README.md

```diff
@@ -25,25 +25,27 @@ Rendezvous service is a software service that permits the Device to find its cur
 ### Build Rendezvous service war file
 * To build rendezvous service, execute the following command:
 ```
-$ mvn clean package
+$ mvn clean install
 ```
 
 * To clean files generated by previous build, execute the following command:
 ```
 $ mvn clean
 ```
 
-* To generate java documentation, execute the following command:
+* To generate java documentation, set the following environment variable and then
+execute the following command:
 ```
+$ export JAVA_HOME=path/to/java11
```

Comment:
While mentioning path in this format, it is better to start with a forward slash.

export JAVA_HOME=/path/to/java11